src/adaBase58_test.c

```diff
@@ -0,0 +1,57 @@
+#include <os.h>
+#include <stdbool.h>
+#include "assert.h"
+#include "test_utils.h"
+#include "hex_utils.h"
+#include "adaBase58.h"
+#include "stream.h"
+
+void run_adaBase58_test()
+{
+#define TESTCASE(input_, expected_) \
+	{ \
+	    stream_t si; \
+	    stream_init(&si); \
+	    stream_appendFromHexString(&si, input_); \
+		uint8_t* inputBuffer = stream_head(&si); \
```

Comment:
This is some weird indent here ...

Reply:
its just github problem? in my code editor, it does not show the indent

Reply:
Hmm, maybe some mix of tabs and spaces?